### DISCUSSION TITLE
为什么要忽略Command组合事件

### DISCUSSION THREAD

**Discussion by mx369** [rime/home#1602] [Q&A]
// ignore Command+X hotkeys.
      if modifiers.contains(.command) {
        break
      }

**Comment by LEOYoon-Tsaw** [comment]
會影響大部分程序和系統的快捷鍵

**Reply by mx369** [reply]
1. 这个禁用的不彻底,依然会影响快捷键,加上这个配置才彻底[{accept: "Super+Super_L", send: Escape, when: composing}],不家这个配置,拼音输入的时候还是会影响系统和其他软件的快捷键
2. 而且别的其他输入法支持meta+BackSpace快捷键 Escape ,拦截了,command,怎么实现这个操作呢

**Reply by LEOYoon-Tsaw** [reply]
你自己改改看吧，挺好改的，好用就提pr
